test/kubemark/start-kubemark.sh

```diff
@@ -23,10 +23,15 @@ set -o pipefail
 TMP_ROOT="$(dirname "${BASH_SOURCE}")/../.."
 KUBE_ROOT=$(readlink -e ${TMP_ROOT} 2> /dev/null || perl -MCwd -e 'print Cwd::abs_path shift' ${TMP_ROOT})
 
-source "${KUBE_ROOT}/test/kubemark/skeleton/util.sh"
 source "${KUBE_ROOT}/test/kubemark/cloud-provider-config.sh"
 source "${KUBE_ROOT}/test/kubemark/${CLOUD_PROVIDER}/util.sh"
 source "${KUBE_ROOT}/cluster/kubemark/${CLOUD_PROVIDER}/config-default.sh"
+
+if ls ${CLOUD_PROVIDER} | grep -Fq 'startup'; then
+  source "${KUBE_ROOT}/test/kubemark/${CLOUD_PROVIDER}/startup.sh"
+fi
+
+source "${KUBE_ROOT}/test/kubemark/skeleton/util.sh"
```

Comment:
You shouldn't source this after sourcing `test/kubemark/${CLOUD_PROVIDER}/util.sh` (it'll break things by redefining functions with empty skeleton implementations). Please fix.